charts/app-config-frontend/values.yaml

```diff
@@ -93,6 +93,41 @@ extraEnvVars: []
 #  - name: BEARER_AUTH
 #    value: true
 
+# -- Custom livenessProbe that overrides the default one
```

Comment:
It would be good to clarify that all `livenessProbe` values are disregarded if this custom livenessProbe is defined. Alternatively, just add the `httpGet` command to `livenessProbe` and users can override it, so the yaml is just copied to the deployment?